src/Omega_h_graph.hpp

```diff
@@ -29,8 +29,18 @@ struct Graph {
   LO nedges() const;
 };
 
+/** \brief combine the edges of two graphs that have the same set of vertices */
```

Comment:
I am fine with 'vertices' here. Just a minor thing, as this function is included here as a graph operation, would 'nodes' be better instead of 'vertices'. In our (parallel-compu.+mesh) context both can be misinterpreted. Either way, up to you.

I would go with 'Combine two graphs that have the same set of nodes'

see use case here: https://github.com/SCOREC/omega_h/blob/00131eb0133796f39ff2ecd0962d9b3d14d5887b/src/Omega_h_mesh.cpp#L384

Reply:
Good point.  I'll change it to 'nodes'.  Thank you.